src/core/server/path/index.ts

```diff
@@ -66,7 +66,7 @@ export const config = {
   path: 'path',
   schema: schema.object({
     // getConfigPath returns the path until the kibana.yml config. We need to move 1 level up for the dir
-    config: schema.string({ defaultValue: () => resolve(getConfigPath(), '..') }),
+    configDir: schema.string({ defaultValue: () => resolve(getConfigPath(), '..') }),
     data: schema.string({ defaultValue: () => getDataPath() }),
```

Comment:
Not renaming `data` because that's an actual value that may sit in the config and it would create breaking changes

Reply:
Following the comment in https://github.com/elastic/kibana/pull/51478#discussion_r352691874, I found that, if I rename the `path.data` to `path.dataDir`, it might break some config files.
The `path.config` to `path.configDir` renaming is not that much of an issue because it previously didn’t exist.

One thing I thought about is to rename it to `path.dataDir` when exposing it in the context, but I’m not sure if that will cause some confusion (if a legacy plugin is reaching for `path.data`, how do we tell them it’s the same but it has simply be renamed to `path.dataDir` in the exposed context but not in the config file?

Reply:
Agreed on slack to rename it while exposing it